Stanford/Jumpstart/Install/VersionIdentifier.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Stanford\Jumpstart\Install;
+
+use \ITasks\AbstractInstallTask;
+
+/**
+ * Class AddFeaturesPage.
+ *
+ * @package Stanford\Jumpstart\Install
+ */
+class VersionIdentifier extends AbstractInstallTask {
```

Comment:
@pookmish I moved the version code in to it's own task so that we can add it to other profiles.